protocol/src/main/java/io/zeebe/protocol/intent/WorkflowInstanceIntent.java

```diff
@@ -32,7 +32,10 @@
   CANCEL((short) 11),
 
   UPDATE_PAYLOAD((short) 12),
-  PAYLOAD_UPDATED((short) 13);
+  PAYLOAD_UPDATED((short) 13),
+
+  CATCH_EVENT_TRIGGERING((short) 14),
```

Comment:
I would prefer something like `BOUNDARY_EVENT_TRIGGERING` to have a clearer distinction to the lifecycle of intermediate catch events.

Reply:
The idea of CATCH_EVENT_TRIGGERING is because this could potentially be reused by the event gateway, where the triggers aren't boundary events, so we figured `CATCH_EVENT_` was an accurate enough name. I'm open to other suggestions, but `BOUNDARY_EVENT_` would be a misleading if we do reuse it...

On the other hand, I can switch it to `BOUNDARY_EVENT_` and let whoever implements event gateway rename it at that point if it makes sense.